pkg/wlsopr/wlsopr.go

```diff
@@ -7,16 +7,17 @@ import (
 	"errors"
 	"fmt"
 
-	"k8s.io/apimachinery/pkg/api/resource"
-
 	"github.com/verrazzano/verrazzano-operator/pkg/util"
 	"github.com/verrazzano/verrazzano-wko-operator/pkg/apis/verrazzano/v1beta1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const microOperatorName = "verrazzano-wko-operator"
+// MicroOperatorName is name of the Weblogic micro operator deployment
+const MicroOperatorName = "verrazzano-wko-operator"
```

Comment:
Why did this constant have to be capitalized?

Reply:
I thought it needed to be global.  It does not.  I have reverted to no capitalization.